File: src/main/java/sc02/Q04.java
package sc02;

import java.io.BufferedReader;
import java.io.InputStreamReader;

/*
Input

3
good
Time
Big

Output

doog
emiT
giB

 */
public class Q04 {
    public static void main(String[] args) {
        try (BufferedReader br = new BufferedReader(new InputStreamReader(System.in))) {
            int count = Integer.parseInt(br.readLine());
            for (int i = 0; i < count; i++) {
                String word = br.readLine();
                StringBuilder answer = new StringBuilder();
                char[] chars = word.toCharArray();
                for (int j = chars.length; j > 0; j--) {
                    answer.append(chars[j - 1]);
                }
                System.out.println(answer);
            }
        } catch (Exception ex) {
            ex.printStackTrace();
        }

    }
}

File: src/main/kotlin/sc02/Q04.kt
package jp.co.kopher.sc02

import java.io.BufferedReader
import java.io.InputStreamReader

/*
Input

3
good
Time
Big

Output

doog
emiT
giB

 */

class Q04

fun main() {
    try {
        BufferedReader(InputStreamReader(System.`in`)).use { br ->
            val count = br.readLine().toInt()
            for (i in 0..<count) {
                val word = br.readLine()
                val answer = StringBuilder()
                val chars = word.toCharArray()
                for (j in chars.size downTo 1) {
                    answer.append(chars[j - 1])
                }
                println(answer)
            }
        }
    } catch (ex: Exception) {
        ex.printStackTrace()
    }
}